src/django_oapif/functions.py

```diff
@@ -1,8 +1,11 @@
 from django.contrib.gis.db.models import functions
 from django.db import models
 
+# see https://code.djangoproject.com/ticket/34882#ticket
+# and https://github.com/django/django/pull/17320
 
-class AsGeoJSON(functions.AsGeoJSON):
+
+class AsGeoJSON(functions.GeoFunc):
```

Comment:
```suggestion
class AsGeoJSON(functions.AsGeoJSON):
```
Isn't this slightly more correct, as if this is not fixed for quite some time, we get all other methods (e.g. `to_oracle_sql`) for free?

Reply:
indeed